### ISSUE TITLE
BUG [T3 Stack, Next.js]: Module parse failed: Identifier '_react' has already been declared 

### ISSUE THREAD

**Issue by VladislavSmolyanoy** [wallabyjs/console-ninja#5]
This doesn’t work using the T3 Stack. 

When importing stuff from React instead of using React functionality directly (`import { useState } from "react"` instead of using `React.useState(...)`) following message occurs:
```
Module parse failed: Identifier '_react' has already been declared (15:6)
File was processed with these loaders:
 * ./node_modules/next/dist/compiled/@next/react-refresh-utils/dist/loader.js
 * ./node_modules/next/dist/build/webpack/loaders/next-swc-loader.js
You may need an additional loader to handle the result of these loaders.
| const _link = /*#__PURE__*/ _interopRequireDefault(require("next/link"));
| const _router = require("next/router");
> const _react = require("react");
| const _image = /*#__PURE__*/ _interopRequireDefault(require("next/image"));
````

**Comment by smcenlly** [comment]
FYI: https://create.t3.gg

**Comment by VladislavSmolyanoy** [comment]
Nah I don’t, I use framer motion for my animations and transitions

**Comment by Addalyn** [comment]
I have the same issue, took me while to debug this issue was almost pulling my hair out especially cause i used consol ninja for couple days and today suddenly my code stopped working, tho i did took break for 2 days or so, i had to work but today when i started my dev log suddenly had this error in console

```
error - ./app/Header.js
Module parse failed: Identifier '_react' has already been declared (16:6)
File was processed with these loaders:
 * ./node_modules/next/dist/compiled/@next/react-refresh-utils/dist/loader.js
 * ./node_modules/next/dist/build/webpack/loaders/next-swc-loader.js
You may need an additional loader to handle the result of these loaders.
| const _styles = require("@mui/material/styles");
| const _react = require("next-auth/react");
> const _react = require("react");
| const _alert = /*#__PURE__*/ _interopRequireDefault(require("@mui/material/Alert"));
| const _avatar = /*#__PURE__*/ _interopRequireDefault(require("@mui/material/Avatar"));
```

I baffled could not find the issue i started production worked just fine, build, all works but dev would not work since i had a a separate dir with 'production' site witch was same code and dev worked there, so i deleted my dev folder and reinit repo to it , and all was working until i reloaded my visual studio and was broken again (i wasn't using nor did i remember i had Console Ninja), so at this point i knew had to be my dev environment aka visual code , then i remembered i had console ninja last installed
uninstalled, reloaded visual code and everything works as expected

in my testing first when i uncomented my <head> component that DID load the site back without errors but i would not know where this error would lie at, i don4t think its cause of 'use client' cause when i tested without the headed site loaded with components that also had 'use client' on them

i do use these 2 imports

```
import { signIn, signOut } from "next-auth/react"
import { useEffect, useState } from 'react'
```
When i tested without them i had other errors doing same thing
```
import Menu from '@mui/material/Menu';
import MenuIcon from '@mui/icons-material/Menu';
```
it would say const _menu error....

so i think ninja renames something? and try's to use the last word of a import that's my only guess
i'm about 6 months into coding in javascript so i'm not an expert into it thought i would share the issue,
see if you can figure it out cause it is a handy app when it worked 

Hope it helps you

**Comment by ArtemGovorov** [comment]
@Addalyn Thanks for the detailed report. Just to confirm, are you also using T3 Stack (https://create.t3.gg/)?

**Comment by ArtemGovorov** [comment]
We've set up a sample project generated by `npm create t3-app@latest`, adding `console.log` to `src/index.tsx` (along with an import if `useState`) seems to work fine:

![cn1](https://user-images.githubusercontent.com/979966/206983997-6faae0e5-e44a-4d9b-922f-86561659b39d.gif)

Can anyone observing the same or similar issue spot what are we missing to reproduce it, or are able to provide a sample repo? Once we can reproduce it, we are confident that we can fix it fairly quickly.

**Comment by Addalyn** [comment]
No i don't use that

i'm willing to give one person access to my repo, however this is not a public repo it's also a project that's ongoing
code cannot be shown or leaked it has also .env file in it, yes i know..., but its private for me alone and i like backups of backups :P
but you can easily clone it, npm install, and npm start dev

**Comment by ArtemGovorov** [comment]
@Addalyn If you can provide me (@ArtemGovorov) an access to the project and some steps how to reproduce (ie. where to add `console.log` to get the error), I'm happy to take a look.

**Comment by ArtemGovorov** [comment]
The `Module parse failed: Identifier '_react' has already been declared` error should be fixed in the v0.0.31. After the version update, please stop the Next.js CLI process, delete the `.next` folder and run `npm run dev` (only required once).
